src/plugins/annotator/src/CollocationsSearchAlgorithm.cpp

```diff
@@ -33,7 +35,7 @@ void CollocationsAlgorithm::findN(const QList<CollocationsAlgorithmItem> &items,
     qint64 i = searchRegion.endPos();
     foreach (const CollocationsAlgorithmItem &item, items) {
         foreach (const U2Region &r, item.regions) {
-            assert(searchRegion.contains(r));
+            SAFE_POINT(searchRegion.contains(r), "Items to search in contains nested regions", );
```

Comment:
nit: the change the comment to the opposite: item region is not in the search region.